.github/workflows/ci-cd.yml

```diff
@@ -112,7 +112,16 @@ jobs:
           name: GuardClauses v${{ needs.ci.outputs.latest_version }}
           path: Artifacts\
 
-      - name: Publish to NuGet
+      - name: Filter changes
```

Comment:
Use the logic from this action for the whole CD job

Reply:
Moved the filter changes step after the checkout in order to work